chain/bitcoind_rpc_events.go

```diff
@@ -86,12 +94,26 @@ func newBitcoindRPCPollingEvents(cfg *PollingConfig,
 		cfg.TxPollingIntervalJitter = 0
 	}
 
+	// Create the config for mempool and attach default values if not
+	// configed.
+	mCfg := &mempoolConfig{
+		client: batchClient,
+	}
+
+	if cfg.RPCBatchSize == 0 {
+		mCfg.getRawTxBatchSize = DefaultGetRawTxBatchSize
+	}
+
+	if cfg.RPCBatchInterval == 0 {
+		mCfg.batchWaitInterval = DefaultBatchWaitInterval
+	}
+
```

Comment:
The `getRawTxBatchSize` & `batchWaitInterval` members of `mCfg` don't get set if a non-default value is used